Framework/PythonInterface/plugins/algorithms/IntegratePeaksProfileFitting.py

```diff
@@ -113,21 +105,50 @@ def PyExec(self):
         else:
             strongPeakParams = None #This will not force any profiles
 
-        nTheta = self.getProperty('NTheta').value
-        nPhi = self.getProperty('NPhi').value
         zBG = 1.96
-        mindtBinWidth = self.getProperty('MindtBinWidth').value
         pplmin_frac = self.getProperty('MinpplFrac').value
         pplmax_frac = self.getProperty('MaxpplFrac').value
         sampleRun = self.getProperty('RunNumber').value
+
+        # There are a few instrument specific parameters that we define here.  In some cases,
+        # it may improve fitting to set tweak these parameters, but for simplicity we define these here
+        # The default values are good for MaNDi - new instruments can be added by adding a different elif
+        # statement.
+        # If you change these values or add an instrument, documentation should also be changed.
+        instrumentName = peaks_ws.getInstrument().getFullName()
```

Comment:
Would it not be better to put these parameters in instrument parameter files rather than hard coding them?

Reply:
We plan to move this to the instrument files once the algorithm is more stable. Right now we just put these parameters in the algorithm so that users and beamline staff can test the algorithm and we don't need to keep changing parameter files.  I can move to the parameters files now if you think that's better?

Reply:
I tend to find that these things can get forgotten and then it's less easy for other instruments to use the code - would it be much hassle to switch now?